pom.xml

```diff
@@ -188,6 +188,7 @@
         <snapshotRepository>
             <id>ossrh</id>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+<!--            <url>https://repo.vertispan.com/gwt-snapshot/</url>-->
```

Comment:
i dont think we need a new commented out line

Reply:
I was not sure, If we may go bay to your repo ... ;-) ... remove it.